src/librustc_back/target/aarch64_apple_ios.rs

```diff
@@ -23,6 +23,7 @@ pub fn target() -> Target {
         options: TargetOptions {
             features: "+neon,+fp-armv8,+cyclone".to_string(),
             eliminate_frame_pointer: false,
+            max_atomic_width: 128,
```

Comment:
Just confirming, but this is inherent to aarch64 (e.g. 128-bit atomics are available everywhere?)